PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -189,7 +189,7 @@ def miniAOD_customizeCommon(process):
 	 valueLabels = cms.vstring('pt','emEnergyFraction'),
 	 lazyParser = cms.bool(True) )
     task.add(process.caloJetMap)
-    process.patJets.userData.userFloats.src += [ cms.InputTag("caloJetMap:pt"), cms.InputTag("caloJetMap:emEnergyFraction") ]
+#    process.patJets.userData.userFloats.src += [ cms.InputTag("caloJetMap:pt"), cms.InputTag("caloJetMap:emEnergyFraction") ]
```

Comment:
Hi Slava,
the user floats were only supposed to be a temporary thing for early data commissioning, and I believe nobody is using them in production.
I think it's best to remove them now from the userfloats, and if there is a usecase for going below 20 GeV of raw calo jet pt then we change the configuration of the calo jet slimmer accordingly.